### DISCUSSION TITLE
access context in the data router

### DISCUSSION THREAD

**Discussion by geohuz** [remix-run/react-router#10100] [Q&A] [answered]
In application with global store inside react context, the fetch apis are normally put inside the global store, but it is not possible to access the store by using hooks, is it possible to have a "context" parameter in the **action/loader** function signatures?

**Comment by geohuz** [comment, marked as answer]
I found the solution to directly import the store.

**Reply by carsten-l** [reply]
What is the solution?

**Reply by geohuz** [reply]
Just import initialized root store and use it (I'm using mobx-state-tree)
